[Wrangler] Document multi-worker testing (#6122)
cloudflare/cloudflare-docs: content/workers/wrangler/api.md

{{</tab>}}
{{</tabs>}}

#### Multi-Worker Example

> It's also possible to test Workers that call other Workers.

Author: ```suggestion
It's also possible to test Workers that call other Workers. In the below example, we refer to the Worker that calls other Workers as the "parent" Worker, and the Worker being called as a "child" Worker.
```

> It's also possible to test Workers that call other Workers. In the below example, we refer to the Worker that calls other Workers as the "parent" Worker, and the Worker being called as a "child" Worker.

Reviewer: ```suggestion
You can test Workers that call other Workers. In the below example, we refer to the Worker that calls other Workers as the parent Worker, and the Worker being called as a child Worker.
```

> Note: if you shut down the child worker prematurely, the parent worker won't know the child worker exists and your tests will fail.

Author: ```suggestion
Note: if you shut down the child Worker prematurely, the parent Worker won't know the child Worker exists and your tests will fail.
```

> Note: if you shut down the child Worker prematurely, the parent Worker won't know the child Worker exists and your tests will fail.

Reviewer: ```suggestion
If you shut down the child Worker prematurely, the parent Worker will not know the child Worker exists and your tests will fail.
```
